proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,452 @@
+# Improvements to the AWS CLI Contribution Process
+
+Proposal         | Metadata
+---------------- | -------------
+**Author**       | Kenneth Daily
+**Status**       | Proposed
+**Created**      | 19-October-2021
+
+## Abstract
+
+This document proposes standards and processes to improve and publish community
+expectations for contributions. It defines the process for proposing,
+submitting, and reviewing a contribution. This proposal will make it easier for
+the community to successfully contribute to the AWS CLI.
+
+## Background and Motivation
+
+### Current contribution process
+
+The AWS CLI GitHub repository is available for any user to report bugs, make
+feature requests, and ask guidance questions through GitHub issues. There are
+currently roughly three stages to manage issues in use that are common to open
```

Comment:
Added a short statement about PRs here, because the "Implementation" section has the details about opening PRs.

I used the more vague 'issues' instead of 'contributions' because the triage process addresses the distinction for guidance questions, but that is out of scope of this document, so I agree with the change.